pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -171,6 +245,17 @@ func createClusterClassBasedClusterTest(tkgctlClient tkgctl.TKGClient, deleteClu
 	}
 }
 
+// deleteClusterClassBasedCluster deletes workload cluster
```

Comment:
again, obvious from the name of the function. and since the function is lowercase named, it will never be exposed thus go.docs will not really use  the comment.